nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps

```diff
@@ -32,7 +32,8 @@ ide/db.dataview/external/commons-compress-1.20.jar java/java.disco/external/comm
 # It happens that maven 3.3.9 uses commons-lang-2.6.jar, same as o.apache.commons.lang
 # and commons-io-2.5.jar, same as java.lsp.server
 # These are used independently, different functional goals
-java/java.lsp.server/external/commons-io-2.5.jar java/maven.embedder/external/apache-maven-3.6.3-bin.zip
+# maven 3.8.4 use common-io-2.6 java/java.lsp.server/external/commons-io-2.5.jar java/maven.embedder/external/apache-maven-3.6.3-bin.zip
+platform/o.apache.commons.io/external/commons-io-2.6.jar java/maven.embedder/external/apache-maven-3.8.4-bin.zip
```

Comment:
The alternative would be to add a dependency on the netbeans module. `maven.embedder` already seems to depend on `commons-logging` (this could most probably be removed)

Reply:
not sure to get the point.

Reply:
Maven embedder could use the NetBeans wide installation of commons-io, but this might not even fix the reported error. Anyway - ignore this.